src/Pretty/Common.hs

```diff
@@ -114,6 +114,10 @@ prettyPrdCns Cns = "cns"
 -- Kinds
 ---------------------------------------------------------------------------------
 
+instance PrettyAnn KindedSkolem where 
+  prettyAnn (sk,Nothing) = prettyAnn sk
+  prettyAnn (sk,Just mk) = "(" <> prettyAnn sk <> ":" <> prettyAnn mk <> ")"
```

Comment:
Probably use `parens` function, and use `<+>` instead of `<>` in order to insert spaces.